rust/src/treefile.rs

```diff
@@ -800,6 +800,26 @@ impl Treefile {
         Ok(set.len() != n)
     }
 
+    pub(crate) fn enable_repo(&mut self, repo: &str) -> Result<()> {
+        let repo_list = repo.split(",");
```

Comment:
Hmm; I was thinking this would always take a single repo name.  Something like:
```
let mut repos = self.parsed.base.repos.take().unwrap_or_default();
repos.push(repo);
self.parsed.base.repos = Some(repos);
```
or so?

I think we need to do this based on how the C code is calling it.

Reply:
Ah maybe this can be:

```
let repos = self.parsed.base.repos.get_or_insert_with(|| Vec::new());
repos.push(repo);
```

See https://doc.rust-lang.org/std/option/enum.Option.html#method.get_or_insert_with

Reply:
I see, I found a couple of examples for multiple repos that passed them like --enablerepo=repo1,repo2
For example: https://www.linuxhelp.com/how-to-enable-or-disable-repositories-in-centos

But I also see what you had in mind. Where people do `--enablerepo=repo1 --enablerepo=repo2.`
Like: https://lists.centos.org/pipermail/centos/2006-April/021285.html

Seem both are supported by yum:
```
       --enablerepo=<repoid>
              Temporarily enable additional repositories for the purpose
              of the current dnf command.  Accepts an id, a
              comma-separated list of ids, or a glob of ids. This option
              can be specified multiple times.
```

We implement  --enablerepo per repo and document that for rpm-ostree?

Reply:
Ahhh!  Yes, then indeed we need to do *both*.